packages/features/ee/workflows/lib/reminders/aiPhoneCallManager.ts

```diff
@@ -0,0 +1,339 @@
+}
+
+interface CreateWorkflowReminderAndExtractPhoneArgs {
+  evt: BookingInfo;
+  workflowStepId: number;
+  scheduledDate: dayjs.Dayjs;
+  seatReferenceUid?: string;
+}
+
+interface CreateWorkflowReminderAndExtractPhoneResult {
+  workflowReminder: { id: number; uuid: string | null };
+  attendeePhoneNumber: string;
+}
+
+const createWorkflowReminderAndExtractPhone = async (
+  args: CreateWorkflowReminderAndExtractPhoneArgs
+): Promise<CreateWorkflowReminderAndExtractPhoneResult> => {
+  const { evt, workflowStepId, scheduledDate, seatReferenceUid } = args;
+
+  // 1) Determine attendee phone first (fail early)
+  let attendeePhoneNumber = extractPhoneNumber(evt.responses);
+  if (!attendeePhoneNumber) {
+    const attendeePhone = evt.attendees?.[0]?.phoneNumber;
+    if (attendeePhone) {
+      attendeePhoneNumber = attendeePhone;
+    } else {
+      throw new Error(`No attendee phone number found for workflow step ${workflowStepId}`);
+    }
+  }
+
+  const workflowReminder = await prisma.workflowReminder.create({
+    data: {
+      bookingUid: evt.uid as string,
+      workflowStepId,
+      method: WorkflowMethods.AI_PHONE_CALL,
+      scheduledDate: scheduledDate.toDate(),
+      scheduled: true,
+      seatReferenceId: seatReferenceUid,
+    },
+  });
+
+  return { workflowReminder, attendeePhoneNumber };
+};
+
+interface ScheduleAIPhoneCallArgs {
+  evt: BookingInfo;
+  triggerEvent: WorkflowTriggerEvents;
+  timeSpan: {
+    time: number | null;
+    timeUnit: TimeUnit | null;
+  };
+  workflowStepId: number | undefined;
+  userId: number | null;
+  teamId: number | null;
+  seatReferenceUid?: string;
+  verifiedAt: Date | null;
+}
+
+export const scheduleAIPhoneCall = async (args: ScheduleAIPhoneCallArgs) => {
+  const { evt, triggerEvent, timeSpan, workflowStepId, userId, teamId, seatReferenceUid, verifiedAt } = args;
+
+  if (!verifiedAt || !workflowStepId) {
+    logger.warn(`Workflow step ${workflowStepId} not yet verified or not found`);
+    return;
+  }
+
+  // Get the workflow step to check if it has an agent configured
+  const workflowStep = await prisma.workflowStep.findUnique({
+    where: { id: workflowStepId },
+    select: {
+      agent: {
+        select: {
+          id: true,
+          providerAgentId: true,
+          outboundPhoneNumbers: {
+            select: {
+              phoneNumber: true,
+              subscriptionStatus: true,
+            },
+          },
+        },
+      },
+    },
+  });
+
+  const activePhoneNumbers = workflowStep?.agent?.outboundPhoneNumbers?.filter(
+    (phoneNumber) => phoneNumber.subscriptionStatus === PhoneNumberSubscriptionStatus.ACTIVE
+  );
+
+  if (!workflowStep?.agent) {
+    logger.warn(`No agent configured for workflow step ${workflowStepId}`);
+    return;
+  }
+
+  if (!workflowStep.agent.outboundPhoneNumbers?.length || !activePhoneNumbers?.length) {
+    logger.warn(`No active outbound phone number configured for agent ${workflowStep.agent.id}`);
+    return;
```

Comment:
Imported phone numbers have no `subscriptionStatus` so `!activePhoneNumbers?.length` is true and we return early